README-spacetime.md

```diff
@@ -0,0 +1,60 @@
+# Spacetime profiling
+
+Spacetime is a feature for showing memory usage over time within OCaml programs. It's useful
+for revealing the location of memory leaks.
+
+## Building for Spacetime
+
+To use Spacetime, you need an OCaml compiler built with that feature enabled. There are
+Spacetime-enabled compiler switches within OPAM. For example, there's the switch 
+4.07.1+spacetime, which works with same OPAM packages as vanilla 4.07.1. To build
+with it in your enviroment, install the switch:
+
+* opam switch create 4.07.1+spacetime # or the current OCaml version for building Coda 
+* eval $(opam env)
+
+Then follow the instructions for "Building outside docker" given in README-dev.md.
+
+## Running for Spacetime
+
+Once the Coda executable has been built for Spacetime, you can run it as you would otherwise.
+To obtain Spacetime profiling data, set the environment variable `OCAML_SPACETIME_INTERVAL`
+to the sampling interval, in milliseconds. In practice, we've found that 10 seconds works well:
+
+* export `OCAML_SPACETIME_INTERVAL`=10000
```

Comment:
Can this be in a markdown code block (delimited by triple backticks on a separate line) instead?

Reply:
Ditto for all the other code blocks; not going to comment them all.